Draft up panics if tasks file missing

If `.draft-tasks.toml` is missing, `draft up` will successfully build and push the image, then release the Helm chart, then will try to execute any tasks - if the tasks file is missing, it will panic:

```
$ draft up
Draft Up Started: 'example-go': 01CBT4KENARZP2WJVH3ME4W9ED
example-go: Building Docker Image: SUCCESS ⚓  (1.0006s)
example-go: Pushing Docker Image: SUCCESS ⚓  (8.5636s)
example-go: Releasing Application: SUCCESS ⚓  (3.5577s)
Inspect the logs with `draft logs 01CBT4KENARZP2WJVH3ME4W9ED`
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x1fd2ca7]

goroutine 1 [running]:
main.runPostDeployTasks(0x0, 0xc420211f60, 0x1a, 0xa, 0xc420513380)
        /Users/radu/go/src/github.com/Azure/draft/cmd/draft/up.go:210 +0x37
main.(*upCmd).run(0xc4202c5770, 0x237afd2, 0xb, 0x0, 0x0)
        /Users/radu/go/src/github.com/Azure/draft/cmd/draft/up.go:201 +0x94a
main.newUpCmd.func2(0xc4201898c0, 0x2fcbae0, 0x0, 0x0, 0x0, 0x0)
        /Users/radu/go/src/github.com/Azure/draft/cmd/draft/up.go:105 +0x101
github.com/Azure/draft/vendor/github.com/spf13/cobra.(*Command).execute(0xc4201898c0, 0x2fcbae0, 0x0, 0x0, 0xc4201898c0, 0x2fcbae0)
        /Users/radu/go/src/github.com/Azure/draft/vendor/github.com/spf13/cobra/command.go:698 +0x47a
github.com/Azure/draft/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc4201b7b00, 0xc42000e010, 0xc4201b7b00, 0xc42071df70)
        /Users/radu/go/src/github.com/Azure/draft/vendor/github.com/spf13/cobra/command.go:783 +0x30e
github.com/Azure/draft/vendor/github.com/spf13/cobra.(*Command).Execute(0xc4201b7b00, 0x0, 0x0)
        /Users/radu/go/src/github.com/Azure/draft/vendor/github.com/spf13/cobra/command.go:736 +0x2b
main.main()
        /Users/radu/go/src/github.com/Azure/draft/cmd/draft/draft.go:178 +0x2d
```

--- a/cmd/draft/up.go
+++ b/cmd/draft/up.go
@@ -207,7 +207,7 @@ func (u *upCmd) run(environment string) (err error) {
 }
 
 func runPostDeployTasks(taskList *tasks.Tasks, buildID string) error {
-	if len(taskList.PostDeploy) == 0 {
+	if taskList == nil || len(taskList.PostDeploy) == 0 {
 		return errors.New("No post deploy tasks to run")
 	}
 

